modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJavaSetResultsAndTotalCheck.testjava

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.account.admin.web.internal.dao.search;
+
+import java.util.Objects;
```

Comment:
@TamyrisTorres, is this import really necessary?

Please talk to @nicolas-sss and @michaelmdelcavalcanti to make sure this test only has what is really necessary since, this week, 2 pull requests were closed due to extra unnecessary code inserted in unit tests like this.

Make you apply the guys advises in all unit test code. :wink: